lib/cmds/workflows_cmds/parse.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const { post } = require('../../workflow');
+const print = require('../../print');
+
+exports.command = 'parse <datasetId>';
+exports.desc = 'Parse a workflow and show expected inputs and input types';
```

Comment:
Took me a bit to understand what `parse` meant in this context.  I wonder if `describe` would be a more clear verb?

Reply:
Works for me, I'll update it.  And I threw this on at the last minute just since it's capabilities the service offers that the UI makes good use of, maybe it's not really valuable from a `cli` perspective?

Reply:
For an example, the ashion cwl would return this `{ tarFile: 'File', source: 'string', reference: 'string' }`